qiskit/__init__.py

```diff
@@ -66,9 +66,6 @@
 # user config
 from qiskit import user_config as _user_config
 
-# The qiskit.extensions.x imports needs to be placed here due to the
-# mechanism for adding gates dynamically.
-import qiskit.extensions
```

Comment:
This probably shows my not understanding of the deprecation process, but what happens now if someone calls `from qiskit.extensions import UnitaryGate`? Would this be still temporarily allowed?

Reply:
All imports from `qiskit.extensions` work as before. This line here was needed before, as the extensions module monkey-patched methods to the circuit, which are only available once the module is imported. 

I moved all methods directly onto the circuit, except `squ` and `snapshot`, so those work as before without needing this import. However if a user now wants to call `QuantumCircuit.squ` that doesn't work anymore... I'll add that import so it still works.

Reply:
Fixed in 3e92ab3b0bfc142ab21e8db9df8b88f7c8c57d17